source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet

```diff
@@ -1,12 +1,30 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
-local question(title, description) = {
-  title: title,
+local questionTitle(isProxy) = (
+  if isProxy then {
+    text: 'How does <em>{person_name}</em> usually travel to work?',
+    placeholders: [
+      placeholders.personName(),
+    ],
+  }
+  else 'How do you usually travel to work?'
+);
+
+local questionDescription(isProxy) = (
+  if isProxy then [
+    'Answer for the longest part, <em>by distance</em>, of their current journey to work',
+    'If the <strong>coronavirus</strong> pandemic has affected how they travel to work, select the answer that best describes their <strong>current circumstances</strong>.',
+  ] else [
+    'Answer for the longest part, <em>by distance</em>, of your current journey to work',
+    'If the <strong>coronavirus</strong> pandemic has affected how you travel to work, select the answer that best describes your <strong>current circumstances</strong>.',
```

Comment:
This applies to all text in question descriptions - why is there a mix of `strong` and `em` tags?

Reply:
I've tried just to use strong throughout bust missed these ones

Reply:
I think we should check which one should be used, as there is significance to using `em` over `strong`.